kpatch-build/kpatch-build

```diff
@@ -750,9 +750,13 @@ elif [[ -e "$KERNEL_SRCDIR"/.config ]] && [[ -e "$VERSIONFILE" ]] && [[ "$(cat "
 	echo "Using cache at $KERNEL_SRCDIR"
 
 else
-	if [[ "$DISTRO" = fedora ]] || [[ "$DISTRO" = rhel ]] || [[ "$DISTRO" = ol ]] || [[ "$DISTRO" = centos ]]; then
+	if [[ "$DISTRO" = fedora ]] || [[ "$DISTRO" = rhel ]] || [[ "$DISTRO" = ol ]] || [[ "$DISTRO" = centos ]] || [[ "$DISTRO" = openEuler ]]; then
 
-		echo "Fedora/Red Hat distribution detected"
+		[[ "$DISTRO" = fedora ]] && echo "Fedora distribution detected"
+		[[ "$DISTRO" = rhel ]] && echo "RHEL distribution detected"
+		[[ "$DISTRO" = ol ]] && echo "Oracle Linux distribution detected"
+		[[ "$DISTRO" = centos ]] && echo "Centos distribution detected"
```

Comment:
"CentOS"

Reply:
sorry for this mistake, already fix it now.